src/control-plane/backend/lambda/api/service/quicksight/sql-builder.ts

```diff
@@ -481,7 +489,7 @@ function _buildEventCondition(eventNames: string[], sqlParameters: SQLParameters
     ),
     `);
   }
-  return newSQL;
+  return [newSQL, computedMethodList];
```

Comment:
return an object to improve the code readability 

```ts
return {
   sql: '',
  methodList: []
};
```

Reply:
fixed